src/test/java/org/apache/commons/logging/pathable/ParentFirstTestCase.java

```diff
@@ -145,8 +143,9 @@ public void testPaths() throws Exception {
         // (context, child, parent).
         final ClassLoader systemLoader = ClassLoader.getSystemClassLoader();
         assertNotNull("System classloader is null", systemLoader);
-        assertNotEquals("System classloader has unexpected type", PathableClassLoader.class.getName(),
-                systemLoader.getClass().getName());
+        assertFalse("System classloader has unexpected type",
```

Comment:
What's the difference?

Reply:
in junit 4 _assertTrue_ is part of TestCase and is available when the tests are running. _assertNotEquals_ was coming from a static import of `org.junit.Assert` which was getting a class not fund exception. It could be changed to _assertNotSame_ as that's also part of the TestCase class.